packages/components/src/styles/components/accordion.scss

```diff
@@ -36,75 +59,104 @@ $hds-accordion-item-border-radius: 6px;
     }
   }
 
-  &.hds-accordion-item--contains-interactive {
+  &.hds-accordion-item--contains-interactive,
+  &.hds-accordion-item--is-static {
     box-shadow: var(--token-surface-base-box-shadow);
   }
 }
 
+.hds-accordion-item--type-flush {
+  box-shadow: 0 1px 0 0 var(--token-color-border-primary);
+
+  &:first-child {
+    box-shadow: 0 1px 0 0 var(--token-color-border-primary), 0 -1px 0 0 var(--token-color-border-primary);
+  }
+
+  // we need this specificity to override the focus-ring mixin
+  .hds-accordion-item__button.hds-accordion-item__button--parent-does-not-contain-interactive::before {
+    border-radius: 0;
+  }
+
+  &.hds-accordion-item--does-not-contain-interactive:not(.hds-accordion-item--is-static) > .hds-disclosure-primitive__toggle {
+    &:hover,
+    &.mock-hover {
+      .hds-accordion-item__toggle {
+        background: var(--token-color-surface-interactive-hover);
+      }
+    }
+
+    &:active,
+    &.mock-active {
+      .hds-accordion-item__toggle {
+        background: var(--token-color-surface-interactive-active);
+      }
+    }
+  }
+}
+
 // TOGGLE BLOCK
 
 .hds-accordion-item__toggle {
   position: relative;
   display: flex;
   gap: 12px;
   align-items: center;
-  padding:
-    $hds-accordion-item-padding
-    $hds-accordion-item-padding
-    $hds-accordion-item-padding
-    12px; // by design
+  padding: var(--hds-accordion-item-padding) var(--hds-accordion-item-padding) var(--hds-accordion-item-padding) 12px;
```

Comment:
@majedelass @andgen404, please double-check that all variants should have 12px left padding.

Reply:
Will do. I was told about these changes yesterday and didn't get the chance to operate them. We also discussed some adjustments for the nested flush accordions that I will need to apply in a follow-up commit.

Reply:
Addressed in 3e847d8 – @majedelass @andgen404 please check